docs/modules/ROOT/pages/amazon-ecr.adoc

```diff
@@ -0,0 +1,17 @@
+= Amazon ECR Client
```

Comment:
Add a link to the nav.adoc in the ROOT folder and also add a list item in the root README.md

Reply:
Done